src/rust/ide/src/controller/graph.rs

```diff
@@ -402,7 +402,8 @@ pub struct Handle {
     pub module : Rc<model::synchronized::Module>,
     parser     : Parser,
     id         : Rc<Id>,
-    logger     : Logger,
+    #[allow(missing_docs)]
```

Comment:
I'd put that as pragma for the whole struct instead. Then the code is more readable.